packages/frontend/src/components/TableConfigPanel/ConditionalFormattingList.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button, FormGroup } from '@blueprintjs/core';
+import { Tooltip2 } from '@blueprintjs/popover2';
+import {
+    createConditionalFormattingConfig,
+    fieldId,
+    getVisibleFields,
+} from '@lightdash/common';
+import produce from 'immer';
+import { useCallback, useMemo } from 'react';
+import { useExplorerContext } from '../../providers/ExplorerProvider';
+import { useVisualizationContext } from '../LightdashVisualization/VisualizationProvider';
+import ConditionalFormatting from './ConditionalFormatting';
+import { ConditionalFormattingListWrapper } from './ConditionalFormatting.styles';
+
+const ConditionalFormattingList = ({}) => {
+    const {
+        explore,
+        tableConfig: { conditionalFormattings, onSetConditionalFormattings },
+    } = useVisualizationContext();
+
+    const activeFields = useExplorerContext((c) => c.state.activeFields);
+
+    const visibleActiveNumericFields = useMemo(() => {
+        if (!explore) return [];
+
+        return getVisibleFields(explore)
+            .filter((field) => activeFields.has(fieldId(field)))
+            .filter((field) => field.type === 'number');
+    }, [explore, activeFields]);
+
+    const activeConfigs = useMemo(() => {
+        return conditionalFormattings.filter((config) =>
+            config.target
+                ? visibleActiveNumericFields.find(
+                      (field) => fieldId(field) === config.target?.fieldId,
+                  )
+                : true,
+        );
+    }, [conditionalFormattings, visibleActiveNumericFields]);
+
+    const usedFieldIds = useMemo(() => {
+        return activeConfigs
+            .map((c) => c.target?.fieldId)
+            .filter((f): f is string => !!f);
+    }, [activeConfigs]);
+
+    const handleAdd = useCallback(() => {
+        onSetConditionalFormattings(
+            produce(activeConfigs, (draft) => {
+                draft.push(createConditionalFormattingConfig());
+            }),
+        );
+    }, [onSetConditionalFormattings, activeConfigs]);
+
+    const handleRemove = useCallback(
+        (index) =>
+            onSetConditionalFormattings(
+                produce(activeConfigs, (draft) => {
+                    draft.splice(index, 1);
+                }),
+            ),
+        [onSetConditionalFormattings, activeConfigs],
+    );
+
+    const handleOnChange = useCallback(
+        (index, newConfig) =>
+            onSetConditionalFormattings(
+                produce(activeConfigs, (draft) => {
+                    draft[index] = newConfig;
+                }),
+            ),
+        [onSetConditionalFormattings, activeConfigs],
+    );
+
+    return (
+        <ConditionalFormattingListWrapper>
+            {activeConfigs.map((conditionalFormatting, index) => (
+                <ConditionalFormatting
```

Comment:
Suggestion: have the rules collapsed if there are than one

`isDefaultOpen={activeConfigs.lenght > 1}`

Reply:
good suggestion, I'll make a change 👍